src/pages/api/index.js

```diff
@@ -1,5 +1,14 @@
 import packageJson from '../../package.json'
```

Comment:
I'm not sure if this is an error but I had to change it to `../.../../package.json`. Is this a config error on my part?